PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace PanoptesNetClient
+{
+    public class Project
+    {
+        public string Id { get; set; }
+    }
+
+    public class ApiClient
```

Comment:
I know you are still working on this class, but I just wanted to comment on its general structure. Right now `ApiClient` is very javascripty and functional. As C# is an object oriented language it may be a better idea to implement this class as a `Singleton` which is an object oriented pattern that allows for just one instance of the object to exist. You can read more about it in c# [here](http://csharpindepth.com/Articles/General/Singleton.aspx).

Singletons give you the ability to both store state and control a resource and, in general, fit in better for an object-oriented language

Reply:
I think this is a good idea. I have read a few articles with people warning against multiple client instances being created and clogging up the app, so I think this would be a good approach to ensure that doesn't occur.